x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/aggs/categorization/CategorizeTextAggregationBuilder.java

```diff
@@ -153,6 +153,13 @@ public CategorizeTextAggregationBuilder setSimilarityThreshold(int similarityThr
     }
 
     public CategorizeTextAggregationBuilder setCategorizationAnalyzerConfig(CategorizationAnalyzerConfig categorizationAnalyzerConfig) {
+        if (this.categorizationAnalyzerConfig != null) {
```

Comment:
Shouldn't we be checking against the filters?

Reply:
The `filters` create an analyzer config, we don't store them.

Reply:
Ah, right. Got it.